apps/utils/locust_tests/requirements.in

```diff
@@ -7,3 +7,4 @@ flask==2.3.3
 requests==2.31.0
 gevent==23.9.1
 urllib3==1.26.18
+werkzeug==3.0.1
```

Comment:
Could you adjust your editor to remove the no new line at EOL warning?

https://thoughtbot.com/blog/no-newline-at-end-of-file#following-the-rules-in-your-editor

Reply:
💯

Reply:
Updated. Thanks